### PR TITLE
npm update and versions bumps

### PR DESCRIPTION
**Pull request by ederign**
## Description
- Regenerated package-lock.json due to node major version bump.
- Removed victory types 
-  Bump @typescript-eslint/eslint-plugin
- Bump   @typescript-eslint/parser

## How Has This Been Tested?

npm install
npm run start:dev

## Merge criteria:

- All the commits have been [_signed-off_](https://github.com/kubeflow/community/tree/master/dco-signoff-hook#signing-off-commits)  (To pass the `DCO` check)

- [X] The commits have meaningful messages; the author will squash them after approval or in case of manual merges will ask to merge with squash.
- [X] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [X] The developer has manually tested the changes and verified that the changes work.
- [X] Code changes follow the [kubeflow contribution guidelines](https://www.kubeflow.org/docs/about/contributing/).

No ui changes.

### CODE REVIEW

File: clients/ui/frontend/.eslintrc [hunk 1]
```diff
@@ -72,7 +72,6 @@
     "no-only-tests/no-only-tests": "error",
     "@typescript-eslint/default-param-last": "error",
     "@typescript-eslint/dot-notation": ["error", { "allowKeywords": true }],
-    "@typescript-eslint/lines-between-class-members": ["error", "always"],
```
Comment (ederign) [inline clients/ui/frontend/.eslintrc:75]: @Griffin-Sullivan https://eslint.org/blog/2023/10/deprecating-formatting-rules/

not sure if we should add something in Prettier to cover that.

File: clients/ui/frontend/src/__tests__/unit/testUtils/hooks.ts [hunk 2]
```diff
@@ -7,14 +7,8 @@ import {
 } from '@testing-library/react';
 import { queries, Queries } from '@testing-library/dom';
 
-export type BooleanValues<T> = T extends
```
Comment (ederign) [inline clients/ui/frontend/src/__tests__/unit/testUtils/hooks.ts:10]: prettier asked for one line.

File: clients/ui/frontend/src/__tests__/unit/testUtils/hooks.ts [hunk 3]
```diff
@@ -7,14 +7,8 @@ import {
...
-  | boolean
-  | number
-  | string
-  | null
-  | undefined
-  // eslint-disable-next-line @typescript-eslint/ban-types
-  | Function
+// eslint-disable-next-line @typescript-eslint/no-unsafe-function-type
```
Comment (ederign) [inline clients/ui/frontend/src/__tests__/unit/testUtils/hooks.ts:10]: as this is a test, I just ignore the warning. @Griffin-Sullivan please let me know if we should refactor it instead.

File: clients/ui/frontend/src/__tests__/unit/testUtils/hooks.ts [hunk 4]
```diff
@@ -174,7 +168,7 @@ export const createComparativeValue = <T>(source: T, booleanTarget: BooleanValue
 
 const createComparativeValueRecursive = <T>(
   source: unknown,
-  // eslint-disable-next-line @typescript-eslint/ban-types
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-function-type
```
Comment (ederign) [inline clients/ui/frontend/src/__tests__/unit/testUtils/hooks.ts:171]: same as above

File: clients/ui/frontend/src/components/browserStorage/BrowserStorageContext.tsx [hunk 5]
```diff
@@ -122,7 +122,7 @@ export const BrowserStorageContextProvider: React.FC<BrowserStorageContextProvid
         setValues((oldValues) => ({ ...oldValues, [storageKey]: storageValue }));
 
         return true;
-      } catch (e) {
```
Comment (ederign) [inline clients/ui/frontend/src/components/browserStorage/BrowserStorageContext.tsx:125]: Complaining about unused var

File: clients/ui/frontend/.eslintrc (hunk 1 above)
Comment (Griffin-Sullivan) [reply clients/ui/frontend/.eslintrc:75]: this is a rule I already didn't care for too much any way. For now we can just remove

File: clients/ui/frontend/src/__tests__/unit/testUtils/hooks.ts (hunk 3 above)
Comment (Griffin-Sullivan) [reply clients/ui/frontend/src/__tests__/unit/testUtils/hooks.ts:10]: fine with me

**Review by Griffin-Sullivan — COMMENTED** [review]
Everything looks good to me. When I have some time tomorrow morning I'm gonna try installing this branch and see how it looks on install. Thanks for doing this Eder!

**Review by Griffin-Sullivan — APPROVED** [review]
Install worked fine for me. Let's merge this one.

**Pull request merged**